packages/api/src/validations/validations.ts

```diff
@@ -143,7 +143,9 @@ interface PresenceValidatorOptions extends WithOptionalMessage {
   allowEmptyString?: boolean
 }
 
-interface UniquenessValidatorOptions extends WithRequiredMessage {}
+interface UniquenessValidatorOptions extends WithOptionalMessage {
```

Comment:
I want to point out here that is a change in behavior. I made `message` optional. It felt weird for it to be required if you just wanted to use the `prismaClient` option and not the `message` option.

I'm not sure what the ramifications of that might be, but it feels right to me. It allowed me to annotate the `let options` instantiation with an actual type :).

Reply:
Is there a default message to fall back to if you don't provide a custom one?

Reply:
@AlejandroFrias Just want to make sure this doesn't blow up if you don't include a message!

Reply:
Int 5919716 I updated the test to make sure the default message is still being respected even when no message option is passed.